shared/search/user-search/search-bar.desktop.js

```diff
@@ -41,32 +41,49 @@ class ServiceIcon extends Component<void, ServiceIconProps, ServiceIconState> {
   }
 }
 
-class SearchBar extends Component<void, Props, void> {
-  _onDebouncedSearch: (overridePlatform?: SearchPlatforms) => void;
+type State = {
+  overridePlatform: ?SearchPlatforms,
```

Comment:
over overridePlatform into state instead of passing it around. Having this as the first argument to search is pretty confusing